MidasCivil_Engine/Convert/ToBHoM/Properties/ToConstraint6DOF.cs

```diff
@@ -119,10 +120,11 @@ public static Constraint6DOF ToConstraint6DOF(this string support, string versio
                                     }
                                 }
                             }
+                            support1 = delimitted[21].Trim();
```

Comment:
`supportName = delimitted[15].Trim(); `
For version = 8.8.1 or prior the name field index is 15.